kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -96,6 +96,35 @@
   const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
+    $trNameSpace: 'user-edit-modal',
+    $trs: {
+      modalTitle: 'Edit Account Info',
+      // input labels
+      fullName: 'Full Name',
+      username: 'Username',
+      userKind: 'User Kind',
+      enterNewPw: 'Enter new password',
+      confirmNewPw: 'Confirm new password',
```

Comment:
By default (where we don't have our own conventions) I believe we decided to follow Material Design conventions.

Their guide uses sentence case: https://material.io/guidelines/style/writing.html#writing-capitalization-punctuation